test/CMakeLists.txt

```diff
@@ -173,6 +173,7 @@ add_compute_test("random.bernoulli_distribution" test_bernoulli_distribution.cpp
 add_compute_test("random.discrete_distribution" test_discrete_distribution.cpp)
 add_compute_test("random.linear_congruential_engine" test_linear_congruential_engine.cpp)
 add_compute_test("random.mersenne_twister_engine" test_mersenne_twister_engine.cpp)
+add_compute_test("random.threefry" test_threefry.cpp)
```

Comment:
Just for consistency, can you name this test `random.threefry_engine` and the source file `test_threefry_engine.cpp`?